### PR TITLE
setup ulimit

### PR DESCRIPTION
**Pull request by acidsennui**
@dincho

### CODE REVIEW

**Comment by dincho** [comment]
context: https://www.pivotaltracker.com/story/show/155932652

File: ansible/setup.yml [hunk 1]
```diff
@@ -84,6 +84,13 @@
       tags:
         - dev-tools
 
+    - name: Increase ulimit
+      pam_limits:
+        domain: epoch
```
Comment (lucafavatella) [inline ansible/setup.yml]: Shall this rather be `"{{ project_user }}"`?

File: ansible/setup.yml [hunk 2]
```diff
@@ -84,6 +84,13 @@
...
+        limit_type: soft
+        limit_item: nofile
+        value: 24576
```
Comment (lucafavatella) [inline ansible/setup.yml]: I see this is taken from https://github.com/aeternity/epoch/blob/v0.9.0/hooks/pre_start.sh#L3
Ok.

File: ansible/setup.yml [hunk 3]
```diff
@@ -84,6 +84,13 @@
       tags:
         - dev-tools
 
+    - name: Increase ulimit
+      pam_limits:
+        domain: epoch
+        limit_type: soft
```
Comment (lucafavatella) [inline ansible/setup.yml]: Can you please confirm that this step fails if hard limit is lower so this soft limit is ineffective?

File: ansible/setup.yml (hunk 1 above)
Comment (acidsennui) [reply ansible/setup.yml]: good catch

File: ansible/setup.yml (hunk 2 above)
Comment (acidsennui) [reply ansible/setup.yml]: yes

**Pull request merged**